cloudhub-2/modules/ROOT/pages/ps-config-app-level-egress.adoc

```diff
@@ -0,0 +1,160 @@
+
+To delete an existing custom rule group: 
+
+. Select *Delete* from the *More actions* menu image:more-icon.png[More actions menu, 2%,2%]. 
+. When prompted, click *Delete Rule Group* to confirma the deletion.
++
+[NOTE]
+Once you delete a rule group, any applications using it immediately starts using the default egress rule group instead.
+
+image::ps-egress-delete-ruleset.png[Delete custom egress rule group]
+
+You can disable the application level egress rule feature by clicking the feature's toggle feature. When prompted, click *Delete and Disable App-Level Rules* to confirm. 
+
+[NOTE]
+Once you disable the feature, all the custom app-level egress rule groups are deleted, and all app-level egress is allowed for all applications.
+
+== Apply Application Level Egress Rules When Deploying Your Applications
+
+You can apply the app-level egress rules that you configured in your private space when you deploy an application:
+
+. When deploying an application to your private space, go to the *Firewall Rules* tab.
++
+The *Egress Rule Group* section shows all the rule groups configured for the selected private space.
+. Use the dropdown to select the egress ruleset you want to apply to your application. 
++
+The selected rule group's details appear below the *Egress Rule Group* list.
+. Click *Apply Changes* to apply the selected egress rules to your application.
+
+
+== Configure Application Egress Connection to Anypoint MQ or ObjectStore v2
+
+To set up connections to Anypoint MQ or OSv2, and access external endpoints, such as `anypoint.mulesoft.com` , you can follow the steps provided in this section.
+
+[NOTE]
+You must also allow outbound access to all 443 at the private space level separately.
+
+To allow HTTPS connections to OSv2, add an app-level egress rule group with the following rule:
+
+* Protocol: `HTTPS`
+* Destination: `[Custom Domains]`
+** With value
+*** For prod: `anypoint.mulesoft.com`
+*** For prod-eu: `eu1.anypoint.mulesoft.com`
+* Port Range: `443`
+
+To allow HTTPS connections to Anypoint MQ, add an app-level egress rule group with the following two rules:
+
+. Allow egress to Anypoint control plane:
+** Protocol: `HTTPS`
+** Destination: `[Custom Domains]`
+*** With value
+**** For prod: `anypoint.mulesoft.com`
+**** For prod-eu: `eu1.anypoint.mulesoft.com`
+** Port Range: `443`
+. Allow egress to MQ destination:
+** Protocol: HTTPS
+** Destination: `[Custom Domains]`
+*** With value of the domain name configured in MQ connector URL
++
+For example, if the MQ connector URL is `https://mq-us-east-1.anypoint.mulesoft.com/api/v1/organizations/{{anypointOrgId}}/environments/{{anypointEnvId}}/`,
+use value `mq-us-east-1.anypoint.mulesoft.com`.
+** Port Range 443
+. Allow egress to Object Store v2 destination
+** Protocol: `HTTPS`
+** Destination: `[Custom Domains}`
+*** With value
+**** For prod: `object-store-us-east-1.anypoint.mulesoft.com`
```

Comment:
Similar to MQ, we should add a note that this URL will vary by region i.e., if user wants to access object store in ap-southeast-1, then object store URL to add here will be object-store-ap-southeast-1.anypoint.mulesoft.com